### DISCUSSION TITLE
Italic fonts to use along with Fira Code

### DISCUSSION THREAD

**Discussion by ronakg** [kovidgoyal/kitty#5146] [Q&A]
I really like the Fira Code font, but it doesn't have an italic variant. Is there any font with an italic variant that comes close to the Fira Code in terms of appearance?

Right now Kitty falls back to Menlo for italic, which are not an ideal pairing with Fira Code.

**Comment by Kruemelmann** [comment]
Maybe this reply helps you:) 
https://github.com/mozilla/Fira/issues/38#issuecomment-136099951
or to follow on the state of this you can follow https://github.com/mozilla/Fira/issues/38
